metadata.rb

```diff
@@ -1,14 +1,14 @@
 name              'percona'
 maintainer        'Sous Chefs'
 maintainer_email  'help@sous-chefs.org'
+chef_version      '>= 13.0'
 license           'MIT'
 description       'Installs Percona MySQL client and server'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 source_url        'https://github.com/phlipper/chef-percona'
 issues_url        'https://github.com/phlipper/chef-percona/issues'
 version           '0.16.4'
 
-depends 'build-essential'
```

Comment:
You need this unless you go chef 14+

Reply:
ok, how should we make foodcritic happy?

Reply:
Disable that rule inline until we remove Chef 13 support